apps/balancer-tools/src/app/apr/api/(utils)/fetchForDateRange.ts

```diff
@@ -96,14 +94,16 @@ const fetchPools = async (
 export async function fetchDataForDateRange(
   startDate: Date,
   endDate: Date,
-  parsedParams: ReturnType<typeof QueryParamsSchema.safeParse>,
+  network: string,
+  maxTvl = 10_000_000_000,
+  minTvl = 10_000,
 ): Promise<{ [key: string]: PoolStatsData[] }> {
-  if (!parsedParams.success) {
-    console.log(parsedParams.error);
-    return {};
-  }
-
-  const existingPoolForDate = await fetchPools(parsedParams);
+  const existingPoolForDate = await fetchPools(
```

Comment:
why fetchPools instead of fetchPoolsFromNetwork?

Reply:
great question! Will try to understand this better, this done on a refactor you did a while ago